zerver/lib/upload/__init__.py

```diff
@@ -137,7 +137,7 @@ def upload_message_attachment(
     file_data: bytes,
     user_profile: UserProfile,
     target_realm: Realm | None = None,
-) -> str:
+) -> tuple[str, str]:
```

Comment:
This is fine, but generally I've been reaching for small dataclasses rather than tuples for this sort of thing, to avoid having a lot of less readable code of the form `function_call()[0]`.